autoscale-core/src/main/java/com/hpe/caf/autoscale/core/ScalerThread.java

```diff
@@ -156,8 +156,37 @@ private void handleAnalysis()
     private void scaleUp(final int amount)
         throws ScalerException
     {
-        LOG.info("Triggering scale up of service {} by amount {}", serviceRef, amount);
+        LOG.info("Attempting scale up of service {} by amount {}", serviceRef, amount);
         scaler.scaleUp(serviceRef, amount);
+        try {
+            InstanceInfo refreshedInsanceInfo = scaler.getInstanceInfo(serviceRef);
+            while (refreshedInsanceInfo.getInstances() > refreshedInsanceInfo.getTotalRunningAndStageInstances()) {
+                boolean instanceMet = false;
+                for (int i = 1; i <= 6; i++) {
+                    final int sleep = i * 10 * 1000;
```

Comment:
I'm not against your defaults but we should probably make these constants configurable.

Reply:
I don't think this is something that we would want to make configurable, its a bit of a cheat to help the code get around some latency we see from the marathon api, not really a feature for a user to configure. This is more in the same spirit as the new one sec wait after a scale down command, we don't really want a user changing it, we are just using to side step slowness in marathon.